packages/ag-charts-community/src/chart/test/examples-gallery.ts

```diff
@@ -5,123 +5,159 @@ import { cartesianChartAssertions, hierarchyChartAssertions, polarChartAssertion
+        enterprise: false,
     },
     SIMPLE_AREA_GRAPH_EXAMPLE: {
         options: examples.SIMPLE_AREA_GRAPH_EXAMPLE,
         assertions: cartesianChartAssertions({ axisTypes: ['time', 'number'], seriesTypes: repeat('area', 4) }),
+        enterprise: false,
     },
     STACKED_AREA_GRAPH_EXAMPLE: {
         options: examples.STACKED_AREA_GRAPH_EXAMPLE,
         assertions: cartesianChartAssertions({ axisTypes: ['time', 'number'], seriesTypes: repeat('area', 6) }),
+        enterprise: false,
     },
     ONE_HUNDRED_PERCENT_STACKED_AREA_GRAPH_EXAMPLE: {
         options: examples.ONE_HUNDRED_PERCENT_STACKED_AREA_GRAPH_EXAMPLE,
         assertions: cartesianChartAssertions({ seriesTypes: repeat('area', 7) }),
+        enterprise: false,
     },
     AREA_GRAPH_WITH_NEGATIVE_VALUES_EXAMPLE: {
         options: examples.AREA_GRAPH_WITH_NEGATIVE_VALUES_EXAMPLE,
         assertions: cartesianChartAssertions({ seriesTypes: repeat('area', 5) }),
+        enterprise: false,
     },
     MARKET_INDEX_TREEMAP_GRAPH_EXAMPLE: {
         options: examples.MARKET_INDEX_TREEMAP_GRAPH_EXAMPLE,
         assertions: hierarchyChartAssertions(),
+        enterprise: true,
     },
     SIMPLE_HISTOGRAM_CHART_EXAMPLE: {
         options: examples.SIMPLE_HISTOGRAM_CHART_EXAMPLE,
         assertions: cartesianChartAssertions({ axisTypes: ['number', 'number'], seriesTypes: ['histogram'] }),
+        enterprise: true,
     },
     HISTOGRAM_WITH_SPECIFIED_BINS_EXAMPLE: {
         options: examples.HISTOGRAM_WITH_SPECIFIED_BINS_EXAMPLE,
         assertions: cartesianChartAssertions({ axisTypes: ['number', 'number'], seriesTypes: ['histogram'] }),
+        enterprise: true,
     },
     XY_HISTOGRAM_WITH_MEAN_EXAMPLE: {
         options: examples.HISTOGRAM_WITH_SPECIFIED_BINS_EXAMPLE,
         assertions: cartesianChartAssertions({ axisTypes: ['number', 'number'], seriesTypes: ['histogram'] }),
+        enterprise: true,
     },
     GROUPED_CATEGORY_AXIS_EXAMPLE: {
         options: examples.GROUPED_CATEGORY_AXIS_EXAMPLE,
         assertions: cartesianChartAssertions({ axisTypes: ['groupedCategory', 'number'], seriesTypes: ['bar'] }),
+        enterprise: false,
     },
     CROSS_LINES_EXAMPLE: {
         options: examples.CROSS_LINES_EXAMPLE,
         assertions: cartesianChartAssertions({ seriesTypes: repeat('bar', 2) }),
+        enterprise: false,
     },
     ADV_TIME_AXIS_WITH_IRREGULAR_INTERVALS: {
         options: examples.ADV_TIME_AXIS_WITH_IRREGULAR_INTERVALS,
         assertions: cartesianChartAssertions({
             axisTypes: ['time', 'number'],
             seriesTypes: ['line', 'line', 'line', 'line'],
         }),
+        enterprise: false,
     },
 };
+
+export const EXAMPLES = Object.entries(COMMUNITY_AND_ENTERPRISE_EXAMPLES)
+    .filter(([, v]) => !v.enterprise)
+    .reduce((pv, [k, v]) => {
+        pv[k] = v;
+        return pv;
+    }, {} as typeof COMMUNITY_AND_ENTERPRISE_EXAMPLES);
```

Comment:
```suggestion
export const EXAMPLES = Object.entries(COMMUNITY_AND_ENTERPRISE_EXAMPLES)
    .filter(([, v]) => !v.enterprise)
    .reduce<typeof COMMUNITY_AND_ENTERPRISE_EXAMPLES>((pv, [k, v]) => {
        pv[k] = v;
        return pv;
    }, {});
```
nit: I know it's just a test, so it's okay if we're less strict here, but generally I would avoid using TS `as` if possible to keep the type checks in place.

Reply:
Nice suggestion, thanks. 100% agree that `as` is a smell, hadn't realised that overriding the inferred type for a `reduce()` call was possible, there are probably a bunch of places we can apply this fix.